package/gluon-status-page/files/lib/gluon/status-page/www/cgi-bin/status

```diff
@@ -22,9 +22,30 @@ function neighbours(ifname)
   for _, line in ipairs(util.split(info)) do
     local data = json.decode(line)
     if data then
-      if data["network"] and data["network"]["mesh_interfaces"] then
-        for _, mac in ipairs(data["network"]["mesh_interfaces"]) do
-          macs[mac] = data
+      if data["network"] then
+        if data["network"]["mesh_interfaces"] then
+          for _, mac in ipairs(data["network"]["mesh_interfaces"]) do
+            macs[mac] = data
+          end
+        end
+        if data["network"]["mesh"] and data["network"]["mesh"]["bat0"] and
+           data["network"]["mesh"]["bat0"]["interfaces"] then
+          local interfaces = data["network"]["mesh"]["bat0"]["interfaces"]
+          if interfaces["other"] then
+            for _, mac in ipairs(interfaces["other"]) do
+              macs[mac] = data
+            end
+          end
+          if interfaces["wireless"] then
+            for _, mac in ipairs(interfaces["wireless"]) do
+              macs[mac] = data
+            end
+          end
+          if interfaces["tunnel"] then
+            for _, mac in ipairs(interfaces["tunnel"]) do
+              macs[mac] = data
+            end
+          end
```

Comment:
This could be shortened a bit by adding a helper where `data` is in scope, as there are four very similar pieces of code:

    local function add_macs(list)
      if list then
        for _, mac in ipairs(list) do
          macs[mac] = data
        end
      end
    end